itchiodl/game.py

```diff
@@ -12,6 +13,12 @@ class Game:
     """Representation of a game download"""
 
     def __init__(self, data):
+        self.args = argv[1:]
+        if '-h' in self.args or '--human-folders' in self.args:
```

Comment:
It shouldn't use -h, will get confused with help

Reply:
Should be good now